### DISCUSSION TITLE
Multi room?

### DISCUSSION THREAD

**Discussion by mesoblaste** [hassio-addons/app-aircast#130] [Q&A] [answered]
I just got Homeassistant and the aircast plugin installed. I wonder if it's supposed to allow broadcasting on multiple devices?
It works on a google home mini but when I click on a HomePod mini it stops playing on the google home and if I click back on google home it only plays on this single device...
Thanks for your help

**Comment by frenck** [comment, marked as answer]
No, it cannot do that

**Reply by mesoblaste** [reply]
OK, thanks for your answer and thanks for your work.

**Reply by kc-nucleon** [reply]
Great add-on, what is it that stops you grouping this real airplay devices if it is all even possible?

**Reply by Se7enair** [reply]
Multiroom with Airplay needs Airplay 2. This addon supports only Airplay 1.

https://github.com/philippe44/AirConnect/issues/539
